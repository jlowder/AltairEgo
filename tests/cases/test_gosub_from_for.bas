10 PRINT "TEST: GOSUB FROM FOR LOOP"
20 FOR I = 1 TO 3
30 PRINT "IN LOOP, I="; I
40 GOSUB 100
50 NEXT I
60 PRINT "TEST PASSED"
70 END
100 PRINT "  IN SUBROUTINE"
110 RETURN
