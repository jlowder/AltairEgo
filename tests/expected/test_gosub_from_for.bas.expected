Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
TEST: GOSUB FROM FOR LOOP
IN LOOP, I= 1
  IN SUBROUTINE
IN LOOP, I= 2
  IN SUBROUTINE
IN LOOP, I= 3
  IN SUBROUTINE
TEST PASSED
OK
